### PR TITLE
AKS Migration changes

### PR DESCRIPTION
**Pull request by adusumillipraveen**
- Renames the component to not override existing app.
- Changes to run with normal withPipeline
- Dockerized the function using https://docs.microsoft.com/en-us/azure/azure-functions/functions-kubernetes-keda
- Added Helm chart using chart-function.
- Removed old infrastructure ( actual app won't be deleted as it's renamed) 
- Uses ServiceBus and blob storage charts for preview.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
+    AzureFunctionsJobHost__Logging__Console__IsEnabled=true
+
+RUN groupadd -r ${GROUP} &&\
+    useradd -r -g ${GROUP} -d / -s /sbin/nologin -c "Docker image user" ${USERNAME}
+
+COPY --chown=${USERNAME}:${GROUP} . /home/site/wwwroot
+
+RUN cd /home/site/wwwroot && \
+    npm install --production
```
Comment (timja) [inline Dockerfile:16]: shouldn't it be yarn?
```suggestion
    yarn install --production
```

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -1,69 +1,13 @@
...
-        def uri = "${profile.userName}:${profile.userPWD}@${profile.publishUrl}"
-        sh "bash uploadSource.sh ${serviceName} functions https://\\${uri}"
-    }
-}
-
-def yarn(task) {
-    sh "cd functions"
-    sh("yarn ${task}")
-    sh "cd .."
+@Library("Infrastructure@removesonar")
```
Comment (timja) [inline Jenkinsfile_CNP:3]: no sonar?

Comment (adusumillipraveen) [reply Jenkinsfile_CNP:3]: Was done temporary as master needs to be run first time and couldn't run it (idea was not to rename before) . Will remove it before emerging.

File: Dockerfile (hunk 1 above)
Comment (adusumillipraveen) [reply Dockerfile:16]: Forgot about this, will try to install yarn, it was giving all kinds of errors when i tried last time.

**Pull request merged**